CorsixTH/Lua/dialogs/resizables/options.lua

```diff
@@ -349,18 +349,6 @@ function UIOptions:buttonHotkey()
   self.ui:addWindow(window)
 end
 
-function UIOptions:buttonBrowseForTHInstall()
-  local function callback(path)
-    local app = TheApp
-    app.config.theme_hospital_install = path
-    app:saveConfig()
-    debug.getregistry()._RESTART = true
-    app.running = false
-  end
-  local browser = UIDirectoryBrowser(self.ui, self.mode, _S.options_window.new_th_directory, "InstallDirTreeNode", callback)
-  self.ui:addWindow(browser)
-end
-
```

Comment:
This is just a deletion for an obsolete function now yes? (I.e. it's now in `folder_settings.lua`)